src/main/java/bio/terra/service/dataset/flight/ingest/IngestLandingFileDeleteAzureStep.java

```diff
@@ -0,0 +1,67 @@
+package bio.terra.service.dataset.flight.ingest;
+
+import bio.terra.model.BillingProfileModel;
+import bio.terra.model.IngestRequestModel;
+import bio.terra.service.common.CommonMapKeys;
+import bio.terra.service.job.JobMapKeys;
+import bio.terra.service.profile.flight.ProfileMapKeys;
+import bio.terra.service.resourcemanagement.azure.AzureContainerPdao;
+import bio.terra.service.resourcemanagement.azure.AzureStorageAccountResource;
+import bio.terra.stairway.FlightContext;
+import bio.terra.stairway.FlightMap;
+import bio.terra.stairway.Step;
+import bio.terra.stairway.StepResult;
+import bio.terra.stairway.exception.RetryException;
+import com.azure.storage.blob.BlobContainerClient;
+import com.azure.storage.blob.BlobUrlParts;
+
+public class IngestLandingFileDeleteAzureStep implements Step {
```

Comment:
Again, thought we had talked about not doing this.

Reply:
Unfortunately we have to